package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "draft-store-client",
+  "version": "1.0.0",
+  "description": "Front end client for draft store",
+  "private": true,
+  "engines": {
+    "node": ">=8.4.0"
+  },
+  "scripts": {
+    "prepublishOnly": "tsc -p ./ --outDir dist/",
+    "lint": "tslint --project tsconfig.json --type-check",
+    "test": "NODE_ENV=mocha LOG_LEVEL=OFF mocha --opts mocha.opts $(find src/test \\( -name '*.ts' \\)) --reporter-options reportFilename=unit,inlineAssets=true,reportTitle=draft-store-client",
```

Comment:
the find isn't needed here just pass the directory or configure it in mocha.opts